### ISSUE TITLE
Subpage sidebar navigation has aria-labelledby issue

### ISSUE THREAD

**Issue by timo54** [ATLAS-Illinois/Illinois-Framework-Theme#1085]
On the Department of Astronomy’s website (https://astro.illinois.edu/), AInspector flags the sidebar-navigation saying, “ARIA attributes that are unsupported or deprecated for a role should be removed. Remove the unsupported aria-labelledby attribute from il-section-nav#block-side navigation element.

<img width="650" height="194" alt="screen capture  of HTML code" src="https://github.com/user-attachments/assets/b7e59b35-af3b-4d90-9abb-e960437c2559" />

Since the code is using aria-labelledby, it should be pointing to an ID “block-sidenavigation-menu” somewhere on the page but such an ID couldn't be found.
